### PR TITLE
Delivery tests

### PR DESCRIPTION
**Pull request by ngaruko**
# Description

[description]

medic/medic-webapp#[number]

# Review checklist

- [x] Readable: Concise, well named, follows the [style guide](https://github.com/medic/medic-docs/blob/master/development/style-guide.md), documented if necessary.
- [x] Documented: Announced in Changes.md in plain English. Configuration and user documentation on [medic-docs](https://github.com/medic/medic-docs/)
- [x] Tested: Unit and/or e2e where appropriate
- [x] Internationalised: All user facing text
- [x] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in Changes.md.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -19,3 +19,4 @@ logs
 
 *.sublime-project
 *.sublime-workspace
+/screnshots/
```
Comment (mandric) [inline .gitignore]: Misspelling?

File: tests/protractor/utils.js [hunk 2]
```diff
@@ -8,8 +8,9 @@ var originalSettings = {};
 
 // The app_settings and update_settings modules are on the main ddoc.
 var mainDdocName = 'medic';
+const userSettingsDocId = `org.couchdb.user:${auth.user}`;
```
Comment (mandric) [inline tests/protractor/utils.js]: Maybe use `var` until the entire module is converted to ES6?

**Comment by mandric** [comment]
You might consider making `tests/protractor/utils.js` ES6 or not ES6 style.  Because having a module with the mixed styles can be confusing.

**Comment by alxndrsn** [comment]
Is this PR a rewrite of https://github.com/medic/medic-webapp/pull/3583?  If so, this seems to have been merged with some similar problems to those discussed on the previous PR.

**Comment by mandric** [comment]
Sorry I was probably a little liberal here.

**Pull request merged**